<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>이미지 자동 분류</string>
  </property>
  <widget class="QPushButton" name="BtnFolderSrc">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>폴더 열기</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>100</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>각 캐릭터마다 진행결과 확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>300</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>캐릭터 우선순위 설정(이동 선택시에만 적용)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>820</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>820</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUP">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>550</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>↑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDown">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>550</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>↓</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>441</width>
     <height>811</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 분류 설정</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 이동 설정</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>340</y>
     <width>261</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QTreeWidget" name="treeWidget_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>590</y>
     <width>261</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="BtnFolderDest">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>결과 폴더 지정</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SrcPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="DestPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>220</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>이동</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>260</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>복사</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>290</y>
     <width>291</width>
     <height>591</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
